### DISCUSSION TITLE
Is there anyone who can give advice on implementing line jumps?

### DISCUSSION THREAD

**Discussion by SWBaek** [excalidraw/excalidraw#8033] [Q&A]
Excalidraw is a great app. As an electronics engineer, I find it extremely useful. 

One feature I wish it had is the ability to automatically create line jumps when two lines intersect, like in Visio or Draw.io. 

I'm trying to implement this myself, but I'm not sure where to start. Could anyone provide advice or recommend any projects I could refer to?

**Comment by dwelle** [comment]
Do you mean something like this?

![image](https://github.com/excalidraw/excalidraw/assets/5153846/db1253bd-8df2-4daa-a378-8bc26f79d614)


If so, that would be pretty hard to do I imagine.

Can you show how it's done in draw.io? Couldn't find it.

**Comment by SWBaek** [comment]
@dwelle 
Exactly!
Are you referring to the source code, or are you asking about how it functions within the app?
I'm a super beginner, so I couldn't find it either...

![image](https://github.com/excalidraw/excalidraw/assets/40167269/5f0ee7c0-6c67-4a31-88a7-169dcb5cd297)
